### DISCUSSION TITLE
DevTalk/Log

### DISCUSSION THREAD

**Discussion by michael-tiger-2010** [RJRock12/MediaView#6] [General]
Developers, talk here about achievements or features.

**Comment by michael-tiger-2010** [comment]
@RJRock12 : I've gotten the communities guidelines to 100% followed. We're good! Also did a bunch of editing and merging and writing and editng and so on. 
Did you know you can open up an online VScode editor by pressing the period key on any `code` tab in the repo?

**Reply by RJRock12** [reply]
@michael-tiger-2010 : I've locked this discussion and reserved it for developers, while the Features and newly created Suggestion Box discussions remain reserved for different types of suggestions. I also created a General Q&A discussion. All of the rules for these discussions have been updated and are listed in the discussions as usual. The goal of all of this was to make suggestions and questions easy to pick out of the different messages. **Ask for approval before changing anything related to the discussions.** (You may have made most of the code, but managing the repo _just so happens_ to be my job.) [[wink.mp4]](https://www.youtube.com/watch?v=mx86-rTclzA)

**Comment by michael-tiger-2010** [comment]
It also just so happens that I know my way around a repo as much as you...

(Changed Wiki)

Can I manage security and pull requests?

**Reply by michael-tiger-2010** [reply]
I'm going to manage the templates, actions, projects tabs. Also going to set some milestones. 

Acting as proofreader for wiki and general overchecker. 

Check Insights tab.

**Reply by RJRock12** [reply]
@michael-tiger-2010 : 

- I’m currently managing security, actions and pull requests, but you can assist. 

- Added code scanning and Dependabot.

- I’m also currently the main developer working on the wiki page. Try not to remove anything, but you can add to it and I’ll proofread for you. I’ll manage most of the wiki, so you shouldn’t need to worry about that.

- You can manage templates and actions by yourself, though.

**Reply by RJRock12** [reply]
(and yes, we probably won’t need Dependabot, but why not?)

**Comment by michael-tiger-2010** [comment]
Had a problem where the Picture in picture wasn't working, found out in chrome videos don't render if they're `display:none` so changed to `width:1px;height:1px` and worked!

**Reply by RJRock12** [reply]
Perfect, did you change the shortcut?

**Reply by RJRock12** [reply]
If so, you need to update the README and/or wiki.

**Reply by michael-tiger-2010** [reply]
Nope accidentally.

**Comment by michael-tiger-2010** [comment]
oh, the bugs with the newest version is overwhelming!

**Reply by michael-tiger-2010** [reply]
fixed most of them

**Comment by michael-tiger-2010** [comment]
The thing where the audio visualizer disappears is supposed to happen.
